lib/init/config-initializer.js

```diff
@@ -345,7 +346,12 @@ function askInstallModules(modules, packageJsonExists) {
  * @returns {Promise<void>} The promise with the result of the prompt
  */
 function promptUser() {
+    const packageJsonExists = npmUtils.checkPackageJson();
 
+    if (!packageJsonExists) {
+        error("Cannot initialize ESLint outside a node project");
+        exit();
```

Comment:
better to give it a non-zero value to state it's not successful.
```suggestion
        exit(1);
```

Reply:
Done